### DISCUSSION TITLE
Keeps saying skip next round.

### DISCUSSION THREAD

**Discussion by pschycoboy** [lmusarella/Pancakeswap-Prediction-Bot-2023#24] [General]
First of all great bot you have really pleased to use it.

But I found a little problem after some rounds mostly close to 7/10 rounds..
The script skips a round but somehow keeps waiting for that round so every other round will be skipped after that. Somehow it breaks itself and needs a restart.
I hope you can consider looking into this 

It also counts the error rounds as a loss in doubling the bet so even a found is not played it still gives the error and raise the bet amount sky high and all of the sudden it starts again 

Hope you have the time because I really like it

**Comment by lmusarella** [comment]
Hello, as I have received many similar reports. This weekend I'll try to take a look, hoping to solve it.

**Reply by pschycoboy** [reply]
Thanks

**Reply by lmusarella** [reply]
Fixed!
